mappings/net/minecraft/world/border/WorldBorder.mapping

```diff
@@ -94,3 +97,39 @@ CLASS net/minecraft/class_2784 net/minecraft/world/border/WorldBorder
 		METHOD <init> (Lnet/minecraft/class_2784;D)V
 			ARG 1 size
 		METHOD method_11996 recalculateBounds ()V
+	CLASS class_5200 Properties
```

Comment:
maybe not properties, doesn't sound like a serialized form of world border to me

Reply:
How is it a serialized form of world border? It's used to serialize it, sure, but that class itself isn't a serialized form, since, well, it's a class

Reply:
`Properties` is an acceptable name. This class is, in essence, an immutable pojo while world border is mutable instead. a world border is converted back and forth to this, and this is written/read to/from nbt.